shotover/src/transforms/cassandra/sink_cluster/rewrite.rs

```diff
@@ -119,7 +119,7 @@ impl MessageRewriter {
                 RewriteTableTy::Prepare { clone_index } => {
                     let mut first = true;
                     for node in pool.nodes().iter() {
-                        if node.is_up && node.rack == self.local_shotover_node.rack {
```

Comment:
Is this going to have a performance impact on non-prepared queries as well? Meaning existing token-aware routing?

Reply:
1. This codepath is only for the routing of prepare requests, it wont affect other message types, and it does not perform token aware routing.
2. Only prepared query execute messages can perform token aware routing.

Reply:
Ok. Thanks.